src/UI/templates/default/Button/button.less

```diff
@@ -0,0 +1,5 @@
+a.btn-link.ilSubmitInactive, a.btn-link.ilSubmitInactive:hover {
+	background-color: inherit;
+	color: @text-color;
+	padding: 0;
+}
```

Comment:
Note, we have some relations here to: https://github.com/ILIAS-eLearning/ILIAS/pull/554. I will adapt this accordingly as soon as accepted/merged.